.setup/distro_setup/ubuntu/bionic/setup_distro.sh

```diff
@@ -89,6 +89,7 @@ update-java-alternatives --set java-1.8.0-openjdk-amd64
 
 # Install Image Magick for image comparison, etc.
 apt-get install -qqy imagemagick
+apt-get install -qqy php-imagick
```

Comment:
move this up with the other `php-*` installs so that they're all consolidated in one place.